target-platforms/packages/create-platform/package.json

```diff
@@ -1,6 +1,9 @@
 {
-	"bin": "./bin/create-platform.js",
-	"description": "Liferay Target Plaform Creator",
+	"bin": {
+		"create-platform": "./bin/create-platform.js",
+		"update-platforms": "./bin/update-platforms.js"
```

Comment:
I think I understand the purpose of this, but just so I can be sure, when would you expect this to be run?

Reply:
I needed it to update all platforms at once whenever I modify the [create-platform script](https://github.com/liferay/liferay-frontend-projects/blob/master/target-platforms/packages/create-plaform/bin/create-platform.js) (for example, when modifying the [base config.json](https://github.com/liferay/liferay-frontend-projects/blob/master/target-platforms/packages/create-plaform/bin/create-platform.js#L32-L73) ).

However, since we have created `portal-base` and rely on sharing code instead of duplicating a generated one (as we did in the begining), we may try to move everything there and remove `create-platform` in the future, maybe...